src/original/ASD_MemorialSloanKettering/MRI-QAMPER_IVIM/utils/nifti_toolbox/license.txt

```diff
@@ -0,0 +1,24 @@
+Copyright (c) 2014, Jimmy Shen
```

Comment:
Should this be included? It seems to be a separate toolbox. https://www.mathworks.com/matlabcentral/fileexchange/8797-tools-for-nifti-and-analyze-image

Reply:
Hey! Thanks for this great submit! 
Some questions:
It seems like the toolbox includes some generic toolboxes like the NIFTI toolbox. Do we need that to be part of the package, or could we take it as a requirement?
Also, what does the code do? Is it just fitting, or does it do more processing?

Thanks!

Oliver and Eric